src/main/java/ru/geekbrains/lesson5/Employee.java

```diff
@@ -0,0 +1,55 @@
+package ru.geekbrains.lesson5;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+public class Employee {
+
+    private String FIO;
+    private String profession;
+    private String email;
+    private String telephone;
+    private int salary;
+    private int age;
+
+    public Employee() {
+        this("Surname Name Fathername", "Profession", "email", "888888888", 0, 18);
+    }
+
+    public Employee(String _FIO, String _profession, String _email, String _telephone, int _salary, int _age) {
+        FIO =_FIO;
+        profession = _profession;
+        email = _email;
+        telephone = _telephone;
+        salary = _salary;
+        age = _age;
```

Comment:
проблему одинакового именования в java решают использованием ссылки на текущий объект `this`, нижнее подчеркивание в начале имени переменной - плохая практика
```suggestion
        this.FIO =FIO;
        this.profession = profession;
        this.email = email;
        this.telephone = telephone;
        this.salary = salary;
        this.age = age;
```